WalletWasabi.Fluent/ViewModels/Dialog/DialogViewModelBase<TResult>.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Reactive.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.Fluent.ViewModels.Dialog
+{
+	/// <summary>
+	/// Base ViewModel class for Dialogs that returns a value back.
+	/// </summary>
+	/// <typeparam name	="TResult">The type of the value to be returned when the dialog is finished.</typeparam>
+	public abstract class DialogViewModelBase<TResult> : DialogViewModelBase
+	{
+		private TaskCompletionSource<TResult>? _tcs { get; set; } 
```

Comment:
As pointed out in https://github.com/zkSNACKs/WalletWasabi/pull/4436#discussion_r497193500, there are two warnings:

1. You shouldn't use `?` here because nullable reference types is not enabled
![1](https://user-images.githubusercontent.com/52379387/94632566-3e31cd80-02cb-11eb-94aa-0fd509aa6f9e.PNG)

2. This is a property and it should be PascalCase (and it should be under the constructors)
![2](https://user-images.githubusercontent.com/52379387/94632591-530e6100-02cb-11eb-9f9d-bc8ff701cef5.PNG)